### PR TITLE
9-2 LynetteFSY Project01 Part 1 

### PR DESCRIPTION
**Pull request by lynettefsy**
Please fill out the survey before submitting the pull request. Thanks!

🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀

How many hours did you spend on this assignment?
I lost count 

Please fill in one error and/or error message you received while working on this assignment.
UNDEFINED , number appear in Console 

What part of the assignment did you spend the most time on?
Trying to figure how to convey the solution into coding. Trial and error

Comfort Level (1-5):
Taking it as 1 - Easy, to 5 - Not Easy
4 cause i got so lost

Completeness Level (1-5):
Yes, completed so what is 1 to 5?
If 1 = not complete and then 5 = complete.
Then i think i complete it.

What did you think of this deliverable?
I do not like the submission, took me awhile to figure it out.

Is there anything in this code that you feel pleased about?
The code freaking works.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,4 +1,124 @@
...
+//   } else if (input == "rock" && comOptions == 2) {
+//     var myOutputValue = "DRAW";
+//   } else if (input == "rock" && comOptions == 0) {
+//     var myOutputValue = "User WINS!";
+//   }
+//   return myOutputValue;
+// };
+
+// Function for AI to have a random Math to select the word
+var aiOptions = function () {
```
Comment (emily-zhangg) [inline script.js:81]: great work extracting your logic intensive code into its own function

File: script.js [hunk 2]
```diff
@@ -1,4 +1,124 @@
...
+    var myOutputValue = "DRAW!";
+  } else {
+    return (myOutputValue = "Please type again, only scissors, paper or rock.");
+  }
+  return myOutputValue;
+};
+
+// Showing them together
+
 var main = function (input) {
```
Comment (emily-zhangg) [inline script.js:121]: great work keeping your main function succinct by putting your game logic in userPick function

File: script.js [hunk 3]
```diff
@@ -1,4 +1,124 @@
...
+  } else if (userOption == "scissors" && comOptions == "paper") {
+    var myOutputValue = "User WINS!";
+  } else if (userOption == "scissors" && comOptions == "rock") {
+    var myOutputValue = "Computer WINS!";
+  } else if (userOption == "paper" && comOptions == "rock") {
+    var myOutputValue = "User WINS!";
+  } else if (userOption == "paper" && comOptions == "scissors") {
+    var myOutputValue = "Computer WINS!";
+  } else if (userOption == comOptions) {
+    var myOutputValue = "DRAW!";
```
Comment (emily-zhangg) [inline script.js:112]: great work being able to use if and else if statements learnt in class here

**Review by yiqiingg — COMMENTED** [review]
great work lynette!!

**Pull request closed without merging**